modules/tiles/src/tileset/tile-3d.ts

```diff
@@ -341,6 +342,24 @@ export class Tile3D {
     this._selectedFrame = 0;
   }
 
+  /**
+   * Memory usage of tile on GPU
+   */
+  _getGpuMemoryUsageInBytes(): number {
+    switch (this.type) {
+      case TILE_TYPE.MESH:
```

Comment:
Sorry, I don't understand this comment, I've maintained the existing behavior for mest&pointcloud which were working fine

Reply:
As `gpuMemoryUsage` != `content.byteLength`, returning `content.byteLength` is not correct for `MESH` because there is also can be a texture. I'd suggest returning 0 here to preserve existing logic for `TILE_TYPE.MESH`

Reply:
I've left if as returning `content.byteLength` as it is more accurate than 0. If there is a reason you think it will break something I can special case it to 0